Sources/Sentry/SentryTracer.m

```diff
@@ -133,6 +138,35 @@ - (void)captureTransaction
     }];
 }
 
+- (void)addMeasurements:(SentryTransaction *)transaction
```

Comment:
m: If I got this right, the first transaction to be transmitted will send appStartMeasurement, but the first transaction may not be UI related or between sessions the first transaction may be different which I think will cause confusion.

Reply:
Good point, @brustolin. We plan to add an app start transaction in the future, which will be the first transaction. Otherwise, currently, you would never get the app start time. We know that this is not optimal, but it's good enough. Or am I wrong, @marandaneto?

Reply:
yep, while we don't have a transaction per app-start, that's what we can do.
I'd even prefer to send such metrics individually, to be honest, but we can't do it as well.